ush/minmon_xtrct_gnorms.pl

```diff
@@ -186,7 +186,7 @@ sub updateGnormData {
 #---------------------------------------------------------------------------
 
 if ($#ARGV != 4 ) {
```

Comment:
Should this be "3" instead of "4" on line 188 now?

Reply:
Good catch. Thank you and fixed.